.changeset/famous-days-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-backend-module-confluence-to-markdown': patch
+---
+
+Added support for Confluence Cloud to the `confluence:transform:markdown` action, view the [README](https://github.com/backstage/backstage/tree/master/plugins/scaffolder-backend-module-confluence-to-markdown) for more details
```

Comment:
Based on the plug-in config , does this only work on Confluence Cloud? Those involved with Atlassian licensing may assume then that this action does not work with a Confluence Server, the on prem solution. Based on the baseUrls and such I feel like maybe it does work on both ? (Though I've not tested it!!)

Reply:
I've updated the wording that this adds Confluence Cloud support along with the existing Confluence Server support.